packages/async-operations/package.json

```diff
@@ -15,7 +15,9 @@
     "tsc": "../../node_modules/.bin/tsc",
     "tsc:listEmittedFiles": "../../node_modules/.bin/tsc --listEmittedFiles",
     "tsc:watch": "../../node_modules/.bin/tsc -w",
-    "watch-test": "../../node_modules/.bin/tsc-watch --onsuccess 'npm test'"
+    "watch-test": "../../node_modules/.bin/tsc-watch --onsuccess 'npm test'",
+    "coverage:validate": "python ../../scripts/coverage_handler/coverage.py --noRerun",
```

Comment:
not big deal, but could you have just one 'coverage' script here and then call it from the root project scripts with the different flag passed in depending on the action?